src/App.php

```diff
@@ -134,42 +134,6 @@ class App
 	 */
 	private $session;
 
-	/**
-	 * @deprecated 2022.03
-	 * @see IHandleUserSessions::isAuthenticated()
-	 */
-	public function isLoggedIn(): bool
-	{
-		return $this->session->isAuthenticated();
-	}
-
-	/**
-	 * @deprecated 2022.03
-	 * @see IHandleUserSessions::isSiteAdmin()
-	 */
-	public function isSiteAdmin(): bool
-	{
-		return $this->session->isSiteAdmin();
-	}
-
-	/**
-	 * @deprecated 2022.03
-	 * @see IHandleUserSessions::getLocalUserId()
-	 */
-	public function getLoggedInUserId(): int
-	{
-		return $this->session->getLocalUserId();
-	}
-
-	/**
-	 * @deprecated 2022.03
-	 * @see IHandleUserSessions::getLocalUserNickname()
-	 */
-	public function getLoggedInUserNickname(): string
```

Comment:
This seems to be used by the calendar, see https://github.com/friendica/friendica/issues/14025#issuecomment-2111605062